client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -987,7 +987,6 @@ WPLineItem.propTypes = {
 	hasDeleteButton: PropTypes.bool,
 	removeProductFromCart: PropTypes.func,
 	product: PropTypes.object.isRequired,
-	getItemVariants: PropTypes.func,
```

Comment:
I don't know how much it matters, but would it make sense to add the new properties here (`allowVariants` and `siteId`) while removing the old ones?

Reply:
Yes, good point. TypeScript is much more reliable than PropTypes, so in general I just stop using PropTypes when code is converted to TS, but that's not really a good reason to let existing ones get out of date; that's especially true here since this file is in `client` which doesn't have active type checking.